config/mora-example.json

```diff
@@ -1,3 +1,10 @@
 {
-    "LORA_URL": "http://mox.example.com/"
+    "LORA_URL": "http://mox.example.com/",
+    "SP_SERVICE_UUIDS": {
+        "service_agreement": "deadbeef-dead-beef-dead-beefdeadbeef",
+        "user": "deadbeef-dead-beef-dead-beefdeadbeef",
+        "user_system": "deadbeef-dead-beef-dead-beefdeadbeef",
+        "service": "deadbeef-dead-beef-dead-beefdeadbeef"
```

Comment:
`00000000-0000-0000-0000-000000000000`? Or perhaps `null`? Also, wouldn't it be nicer to have each of these as a separate setting?

On a totally unrelated note, I really want this to die — we should have used Flasks configuration support instead…

Reply:
The service UUID structure is what the SP library expects as a parameter. It does make more sense to split it up, though. I'll do that.